### PR TITLE
[debug] Print strings using their description in the debugger

### PR DESCRIPTION
**Pull request by Azoy**
When using `po` in the debugger on a string, it currently prints out the debugDescription which should be used when using `p`. Check for string and just return the description.

Resolves: rdar://109902298

### CODE REVIEW

**Comment by Azoy** [comment]
@swift-ci please smoke test

File: stdlib/public/core/DebuggerSupport.swift [hunk 1]
```diff
@@ -148,6 +148,8 @@ public enum _DebuggerSupport {
       return count == 1 ? "1 element" : "\(count) elements"
     case .`struct`?, .`enum`?, nil:
       switch value {
+      case let x as String:
+        return x.description
```
Comment (kastiglione) [inline stdlib/public/core/DebuggerSupport.swift:152]: I was hoping it was as simple as this. Might be worth a comment pointing out that `String.debugDescription` has the unwanted (debugging) side effects of escaping newlines.

**Review by kastiglione — APPROVED** [review]
Thanks!

**Comment by Azoy** [comment]
@swift-ci please smoke test

**Comment by Azoy** [comment]
@swift-ci please smoke test

**Review by kastiglione — APPROVED** [review]
looks great, thanks!

**Pull request merged**